src/Lib/ViewCanvas.vala

```diff
@@ -338,6 +339,11 @@ public class Akira.Lib.ViewCanvas : ViewLayers.BaseCanvas {
 
                     selection_manager.add_to_selection (target.id);
                     selection_manager.selection_modified_external ();
+                } else {
+                    if (selection_manager.selection.count () > 1) {
+                        // Don't trigger sub selection when only one item's selected
+                        nob_manager.toggle_sub_selection (target.id);
+                    }
```

Comment:
I think the sub selection should get triggered only if the user holds the <kbd>ALT</kbd> key.
Clicking on an item that is part of a multi selection should deselect the other items and only select the clicked one. (this is something not implemented yet but we should in a future PR).

Reply:
Coming from Illustrator, I tend to disagree with this behaviour. In Inkscape (which is what I use exclusively today) always relying on the UI select to choose which item to use as anchor is very frustrating. In Illustrator all I had to do was to click on the already selected item to make it become the anchor.

I think this approach is winning and way easier than having to press the ALT key. 

Also, I'm not very used with deselecting all the items is I click on an already selected item. Moreover, I think this could be annoying, imagine I was about to select 30 items one at a time and suddenly, by mistake, I would click on an already selected item with the not so desirable result that my entire selection has gone. 

With the anchor approach, I would have simply "elected" that item to become the anchor. If this is not the case, I could simply click it again and remove the anchoring. If I want to deselect it, I simply need to hold the SHIFT key. 

I think this approach is better. 
Let me know what you think, I can change the code, of course, but this particular feature was something I wanted so badly and I was not able to find in any other software but Illustrator (which is, you know, a good example :)) that I want to point out my thoughts about it.

Reply:
Thanks for the detailed explanation.
This is mostly a UX decision, and my proposal was coming from my past experience with Sketch and Figma.

I see your points and I think they're very valid, so I'm okay with following your suggested approach.